src/variationallaplace/transferfunction.jl

```diff
@@ -18,7 +18,7 @@ function _transferfunction(ω, ∂f∂x, ∂f∂u, ∂g∂x)
         for i = 1:ng
             for k = 1:nk
                 # transfer functions (FFT of kernel)
-                Sk = (1im*2*pi*ω .- Λ[k]).^-1
+                Sk = ((im*2*pi) .* ω .- Λ[k]).^-1
```

Comment:
this decreases the number of allocations to 347640 but no meaningful speed improvement it seems